test/Chainweb/Test/BlockHeader/Validation.hs

```diff
@@ -157,58 +157,58 @@ prop_fail_validate = testCase "validate invalid BlockHeaders" $ do
 validationFailures :: [(TestHeader, [ValidationFailureType])]
 validationFailures =
```

Comment:
Please add an entry that specifically fails for feature flag validation by setting it to an invalid value.